packages/connect-explorer/src/pages/methods/other/init.mdx

```diff
@@ -33,14 +42,27 @@ export const paramDescriptions = {
     popup: 'default `true`. Projects running on trusted domains (trezor.io) are not required to use popup. For other domains this option is ignored.',
```

Comment:
Do we have to mention if it is not available for any external developers?

Reply:
It's better to have it documented regardless imo